website/src/components/ComparisonTable.tsx

```diff
@@ -98,7 +100,7 @@ const toolchainRows: Comparison[] = [
 		feature: 'Supported languages',
 		support: {
 			moon: 'Bash, Batch, Bun, Deno, Go, Node, JavaScript, TypeScript, PHP, Ruby, Rust, Python',
-			nx: 'JavaScript, TypeScript',
+			nx: 'All languages via plugins. OOTB TS/JS, existing plugins for Rust, Go, Dotnet and more',
```

Comment:
@JamesHenry This section is mostly about an integrated toolchain. A plugin system supporting other languages really isn't the same thing.

We may just need another section for this, but I can do that after merging.

Reply:
It's your website, so you are free to structure however you wish. But it is incorrect to imply that Nx only supports JavaScript and TypeScript.

Nx is architected in a similar way to VSCode or other IDEs, we have an entirely agnostic core that deals with project graph construction, task running and caching etc, and then everything else is applied via plugins. The Nx Team provides many such plugins so that things don't need to be reinvented, just like the VSCode team does, but we have a ton from the community as well: https://nx.dev/plugin-registry

This plugability very much includes programming languages, and we have plenty of precedent of mixed language monorepos powered by Nx.

Reply:
I don't disagree, but in moon there's a distinction between toolchain languages, and languages supported via plugins. It's not really reflected right now, so I'll update it after the fact.